src/topology.rs

```diff
@@ -214,4 +214,12 @@ impl Topology {
     pub fn get(&mut self, id: &ed25519::PublicKey) -> Option<&Arc<Profile>> {
         self.profiles.get(id)
     }
+
+    pub fn peers(&self) -> Vec<&Arc<Profile>> {
+        self.profiles.iter().collect()
+    }
```

Comment:
I think it would be more handy for the long term to have the peer returns the `Profiles` instead of the list of all the profiles. That way the users are free to explore the collections they way they see fit.
```suggestion
    pub fn peers(&self) -> &Profiles {
        &self.profiles
    }
```